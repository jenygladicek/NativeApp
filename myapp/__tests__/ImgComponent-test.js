import React, {Component} from 'react';
import  'react-native';
import ImgCombo from './../src/components/ImgComponent';

import renderer from 'react-test-renderer';

describe('Welcome (Snapshot)', () => {
    it('Welcome renders Image', () => {
      let component = renderer.create(<ImgCombo />);
      let json = component.toJSON();
      expect(json).toMatchSnapshot();
    });
  });