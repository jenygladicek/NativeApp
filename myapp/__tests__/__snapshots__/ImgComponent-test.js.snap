// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome (Snapshot) Welcome renders Image 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/images/img1.jpg",
        }
      }
    />
  </View>
</View>
`;
